### PR TITLE
SpeedRunEthereum banner on developers page

### PR DESCRIPTION
**Pull request by carletex**
Hey all!

This PR results from a conversation between @austintgriffith and @minimalsm, @nloureiro, Jakub, etc  (That's why I didn't create an issue first, but happy to do so if needed!).

The idea is to have a SpeedRunEthereum banner on the `/developers` page.

Here is a preview video of how it would look like: 

https://github.com/ethereum/ethereum-org-website/assets/2486142/fd7ff49a-c073-4c8e-8d31-6dfece99cc22

---

I'm not familiar with the codebase, but tried to stick to the project standards. Let me know if there is anything I should change (code or design related), or feel free to push any changes directly.

Thank you!

### CODE REVIEW

**Review by nhsz — APPROVED** [review]
LGTM

**Comment by nloureiro** [comment]
@carletex, thank you for putting this together.

I just have a small adjustment for the mobile, to make it centered and aligned with the other sections
Screen Shot 2024-05-03 08 56 28 AM

File: src/pages/developers/index.tsx [hunk 1]
```diff
@@ -136,6 +139,42 @@ const StyledCallout = chakra(Callout, {
...
+      spacing={{ base: "3", md: "4" }}
+      p={{ base: "6", lg: "8" }}
+      position="absolute"
+      insetInlineStart={{ md: "8" }}
+      maxW={{ base: "lg", xl: "xl" }}
+      top={{ base: "0", md: "50" }}
+      wordBreak="break-word"
+      alignItems="flex-start"
+    >
+      <Heading as="h2" size="xl">
```
Comment (TylerAPfledderer) [inline src/pages/developers/index.tsx:170]: This is actually the default in Chakra, so you do not need to pass props here to render an h2 with `xl` size.

```suggestion
      <Heading>
```

We will also be handling more defaults under the hood soon to minimize the need for passing a value to `size`

**Review by TylerAPfledderer — COMMENTED** [review]
Hey @carletex! I'm a power contributor here and also a collaborator/maintainer for Chakra. Figured I take a quick look here at your approach. 

Nice work with implementation in the code structure. Just one tiny comment from me for consistency sake, but otherwise looks ready to go! 😁

File: src/pages/developers/index.tsx (hunk 1 above)
Comment (carletex) [reply src/pages/developers/index.tsx:170]: oh nice, just saw the component description: 

image

Thanks!!

**Review by nloureiro — APPROVED** [review]

**Review by nloureiro — APPROVED** [review]

File: src/pages/developers/index.tsx [hunk 2]
```diff
@@ -136,6 +139,42 @@ const StyledCallout = chakra(Callout, {
...
+        xl: "auto",
+      }}
+    />
+    <Stack
+      spacing={{ base: "3", md: "4" }}
+      p={{ base: "4", lg: "8" }}
+      position="absolute"
+      insetInlineStart="8"
+      maxW={{ base: "lg", xl: "xl" }}
+      top={{ base: "25", xl: "50" }}
```
[the remark this answers is not in the document]
Comment (minimalsm) [reply src/pages/developers/index.tsx]: As this is a temporary banner for the moment, I'm just going to merge it as is

**Pull request merged**